prow/plugins/trigger/pr.go

```diff
@@ -77,7 +77,7 @@ func handlePR(c client, trustedOrg string, pr github.PullRequestEvent) error {
 func welcomeMsg(ghc githubClient, pr github.PullRequest, trustedOrg string) error {
 	commentTemplate := `Hi @%s. Thanks for your PR.
```

Comment:
We will probably need to make this template configurable :/

Reply:
You tell me what you want me to do, or file an issue for me to be assigned and I'll give it a shot. I've fielded the answer to "how do I join the org" a couple times inside of IBM and out, so I figure others might want the same answer.

Reply:
We run the same code for [openshift/origin](https://github.com/openshift/origin) and ideally we would redirect to the openshift org for origin contributions.

Reply:
Agree with @kargakis .
@MHBauer FYI: this PR https://github.com/kubernetes/test-infra/pull/4491 made a configurable change. Hope it can help something here.

Reply:
An example PR that exposes knobs in the plugin config (https://github.com/kubernetes/test-infra/pull/4714) is probably a better example since this will need to flow through the plugin config.